docs/ref/forms/models.txt

```diff
@@ -1,15 +1,257 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+===========
+ModelForms
+===========
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. class:: ModelForm
+
+.. _ref-modelform:
+
+.. currentmodule:: django.forms
+
+:class:`~django.forms.ModelForm` API reference. For introductory material about
+ModelForms, see the :doc:`/topics/forms/modelforms` topic guide.
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` Options
+==========================
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled by setting metadata options as attributes of an inner
+``Meta`` class.
+
+::
+
+    # Example using some sort of Book model
+
+    from django.forms import ModelForm
+    from myapp.models import Book
+
+
+    class BookForm(ModelForm):
+        class Meta:
+            model = Book
+            fields = ["title", "author"]
+            help_texts = {
+                "title": "The title of the book",
+                "author": "The author of the book",
+            }
+            # ... other attributes
+
+.. admonition:: Invalid Meta class attributes
```

Comment:
Thinking of the flow, I think this can be above the specific attributes but further down (so above `error_messages`). This information is like an aside so I think can come later